src/plugins/footNotePlugin.js

```diff
@@ -6,16 +6,23 @@
  *
  */
 
+function FootNoteCounter(initialCount = 0) {
+  this.count = initialCount;
+
+  FootNoteCounter.prototype.getNextCount = function getNextCount() {
+    this.count = this.count + 1;
+    return this.count;
+  };
+}
+
 export default function createFootnotePlugin() {
-  const embedToHTML = (embed, _, context) => {
-    const footNoteEntryNum = Object.keys(context).length + 1;
+  const metaDataCounter = new FootNoteCounter();
+  const embedToHTMLCounter = new FootNoteCounter();
```

Comment:
Hmm, i og med at disse tellerene opprettes samtidig som fotnote-pluginen vil de vel fortsette å telle oppover helt til komponenten redeployes? Hvis jeg ikke misforstår vil det føre til at fotnote-numrene ikke alltid starter på 1 for alle artikler?

Reply:
Det var derfor jeg gjorde om export'en fra plugins/index.js til funksjon. Derfor jeg nå kaller createPlugins().

Reply:
Aha, ja. Det overså jeg helt. Da skjønner jeg hvordan fungerer som før